### DISCUSSION TITLE
Disable session timeout?

### DISCUSSION THREAD

**Discussion by InputUsername** [kafbat/kafka-ui#516] [Q&A] [answered]
I am trying to serve Kafbat's API for use by internal tools. I know I can get a session token to access the API from these tools, but the session seems to expire after a while.

The docs (http://kafbat.io/configuration/misc-configuration-properties) mention the `server.reactive.session.timeout` setting, but I have not set this. Can I disable session timeout?

Another solution would be disabling authentication entirely, but I can't find anything in the documentation about that either.

Disclaimer: I'm still in the process of migrating from the old version of kafka-ui, but I haven't seen any updates on authentication, so I'm assuming it works the same in Kafbat.

**Comment by Haarolean** [comment, marked as answer]
1. `server.reactive.session.timeout` defaults to 30 minutes. You can set it like `server.reactive.session.timeout=999999999h`.
2. Disabling authentication is just enabling it but backwards. Set `auth.type` to `DISABLED` rather than whatever you have.

**Reply by InputUsername** [reply]
Thanks for the response, disabling auth sounds like the solution for this use case 👍
